### PR TITLE
ci: Hardcode runner-determinator

### PR DESCRIPTION
**Pull request by zxiiro**
Hardcode the runner-determinator script for testing ALI.

### CODE REVIEW

File: .github/workflows/_runner-determinator.yml [hunk 1]
```diff
@@ -34,11 +34,120 @@ jobs:
...
-        with:
-          fetch-depth: 1
-          submodules: true
+      # - name: Checkout PyTorch
+      #   uses: pytorch/pytorch/.github/actions/checkout-pytorch@main
+      #   with:
+      #     fetch-depth: 1
+      #     submodules: true
+
+      - name: Hardcode runner-determinator script
```
Comment (ZainRizvi) [inline .github/workflows/_runner-determinator.yml:43]: How about adding a comment here explaining that this is temporary and that the embedded file should match the python script it was sourced from?

**Review by ZainRizvi — APPROVED** [review]

File: .github/workflows/_runner-determinator.yml (hunk 1 above)
Comment (zxiiro) [reply .github/workflows/_runner-determinator.yml:46]: Good point, I added some comments and a TODO comment too.

**Comment by zxiiro** [comment]
I think the 2 failures are intermittent as this pass previously.

**Pull request closed without merging**